### PR TITLE
Separate registrations

### PR DESCRIPTION
**Pull request by lukanus**
# What 🕵️‍♀️
Separate validator code from the rest of the relay

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/blocknative/dreamboat
 
-go 1.18
+go 1.19
```
Comment (aratz-lasa) [inline go.mod:3]: 👀

**Review by aratz-lasa — APPROVED** [review]
Looks really good! I like a lot the new separation and interfaces

File: pkg/validators/storemanager.go [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+	"time"
+
+	"github.com/blocknative/dreamboat/pkg/structs"
+	"github.com/flashbots/go-boost-utils/types"
+	lru "github.com/hashicorp/golang-lru/v2"
+	"github.com/lthibault/log"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// StoreReqItem is similar to VerifyReq jsut for storing payloads
```
[the remark this answers is not in the document]
Comment (lukanus) [reply pkg/validators/storemanager.go:18]: hmmm I just copy pasted :) I can correct this

File: go.mod (hunk 1 above)
Comment (lukanus) [reply go.mod:3]: @aratz-lasa I had a thought that we'll be testing that anyway :D

**Pull request merged**